core/domain/exp_services.py

```diff
@@ -1017,6 +1293,15 @@ def save_new_exploration_from_yaml_and_assets(
     exp_domain.Exploration.LAST_UNTITLED_SCHEMA_VERSION,
     since in that case the YAML schema will not have a title and category
     present.
+
+    Args:
+        committer_id: str. The is of the user who made the commit.
+        yaml_content: str. The YAML representation of the exploration.
+        exploration_id: str. The id of the exploration.
+        assets_list: list. A list of assets.
```

Comment:
assets_list is a list of lists containing filename & content.so can we write the data type as **list(list(str)).** ?